### ISSUE TITLE
Error when starting an Activities

### ISSUE THREAD

**Issue by dgrivera** [sensepost/objection#250]
**Describe the bug**
When I started one of the activity the objection crash. Kindly see the evidence.

**To Reproduce**
1. List down the activities : android hooking list activities
2. Select Activity and run it : android intent launch_activity com.example.a11x256.frida_test.my_activity

**Expected behavior**
Activity should run when started by Objection

**Evidence / Logs / Screenshots**

```
com.example.a11x256.frida_test on (HUAWEI: 8.1.0) [usb] # android intent launch_activity com.example.a11x256.frida_test.my_activity    
(agent) Starting activity com.example.a11x256.frida_test.my_activity...
An unexpected internal exception has occurred. If this looks like a code related error, please file a bug report!
(session detach message) server-terminated
script is destroyed

Python stack trace: Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/objection/console/repl.py", line 371, in start_repl
    self.run_command(document)
  File "/usr/local/lib/python3.7/site-packages/objection/console/repl.py", line 185, in run_command
    exec_method(arguments)
  File "/usr/local/lib/python3.7/site-packages/objection/commands/android/intents.py", line 22, in launch_activity
    api.android_intent_start_activity(intent_class)
  File "/usr/local/lib/python3.7/site-packages/frida/core.py", line 322, in method
    return script._rpc_request('call', js_name, args)
  File "/usr/local/lib/python3.7/site-packages/frida/core.py", line 250, in _rpc_request
    raise result[2]
frida.InvalidOperationError: script is destroyed

```

```text
objection --debug explore
- [incoming message] ------------------
{
  "payload": "Starting activity \u001b[32mcom.example.a11x256.frida_test.my_activity\u001b[39m...",
  "type": "send"
}
- [./incoming message] ----------------
(agent) Starting activity com.example.a11x256.frida_test.my_activity...
An unexpected internal exception has occurred. If this looks like a code related error, please file a bug report!
script is destroyed
- [incoming message] ------------------
"server-terminated"
- [./incoming message] ----------------
```

**Environment (please complete the following information):**
 - Device: Huawei Nova 3i
 - OS: Android 8.1.0
 - Frida Version 12.6.10
- Objection Version 1.6.6

**Additional context**
Just want to run other activity.

**Comment by leonjza** [comment]
Anything useful in `adb logcat`?
It may be worth it to decompile the app and see what the activity is doing. If it needs extra data, then it is expected to crash (for now, the [implementation](https://github.com/sensepost/objection/blob/master/agent/src/android/intent.ts#L16-L17) assumes no extra data is needed). PR's welcome to add extra data support!

**Comment by dgrivera** [comment]
Previous activity where working last time but now its crashing. No extra data needed to it previously. I was surprised when I re-test the mobile app it crashed.

**Comment by leonjza** [comment]
There definitely was something up with activity launching which I may have fixed now (although I was getting different errors to you). Fix should land in the next release.

**Comment by dgrivera** [comment]
Thank you so much your the best! Hope that things will work again.

**Comment by leonjza** [comment]
This should be fixed in v1.7.0.
